src/main/java/dev/dfonline/codeclient/mixin/world/block/MBlock.java

```diff
@@ -42,4 +62,47 @@ private void getPickStack(WorldView world, BlockPos pos, BlockState state, Callb
                 }
         }
     }
+
+    @Unique
+    private static final Pattern NAME_PICKABLE_CATEGORY_REGEX =
+            Pattern.compile("^([CALSTR]{4,5} )?((FUNCTION)|(PROCESS))$");
```

Comment:
regex is a tiny bit overkill but stuff like this is used elsewhere anyway.
I would do ^(((CALL)? FUNCTION)|((START)? PROCESS))$

Reply:
It totally is, I think I just saw that CommandJump was using regex so I turned my couple if checks into a regex lol
I'll change it